lua/plenary/busted.lua

```diff
@@ -219,16 +275,17 @@ clear = mod.clear
 assert = require("luassert")
 
 mod.run = function(file)
-  print("\n" .. HEADER)
-  print("Testing: ", file)
+  -- print("Testing: ", file)
 
   local ok, msg = pcall(dofile, file)
 
   if not ok then
-    print(HEADER)
+    print(HORIZONTALRULER)
     print("FAILED TO LOAD FILE")
     print(color_string("red", msg))
-    print(HEADER)
+    print(HORIZONTALRULER)
+
+    os.exit(2)
```

Comment:
This shouldn't be here

I am looking at the whole code now. Give me between 30 ~ 60 mins for an answer. I wanna play around with the code for a bit

Reply:
Okay i am trying to get `nvim --headless --noplugin -u scripts/minimal.vim -c "PlenaryBustedFile tests/plenary/simple_busted_spec.lua"` to work first. After that i am trying to deal with `PlenaryBustedDirectory`

Reply:
Oh i see what you doing. I am not sure if i like it i kinda wanna keep the current way of starting a test, with just executing busted itself. We not always wanna go thought `test_harness`

Reply:
That's exactly the reason for reaching out.
I don't like it either.
I'm afk for +/- 2 hours. So will respond later.

Reply:
Here is what i think we could do. When executing the file with `PlenaryBustedFile` we have basically the same output, a little bit cleaned up and for `Directory` we can then get the `SUCCESS`, `FAILURE` and `PENDING` outputs and count them and also convert them to dots.

For stacktrace and error, the line always starts with a ` ` and those always come after a `FAILURE` so, if we read failure we go in another state that captures all further lines that start with ` ` and memorizes them for later output. And on exit we just draw it. Colored dots, Failures and summarize. We also can have `simple_busted` print the summarize and just realize that its a summarize and capture it that way.

If you wanna have a more detailed talk, you can write me on gitter in private.